cmd/crio/main.go

```diff
@@ -325,9 +325,9 @@ func main() {
 			Value:  libkpod.DefaultHooksDirPath,
 			Hidden: true,
 		},
-		cli.StringFlag{
-			Name:   "default-mounts-path",
-			Usage:  "set the default mounts file path",
+		cli.StringSliceFlag{
+			Name:   "default-mounts",
+			Usage:  "set the default mounts paths in the form host:container",
```

Comment:
add a default mount path in the form host:container

Reply:
fixed